README.md

```diff
@@ -42,3 +42,8 @@ Customize this extension via the bpmnJSTracking config:
 | `canvas.palette.click` <br> `canvas.palette.dragstart`| <ul><li>name</li><li>data-action</li><li>selection</li></ul>|
 | `canvas.popupMenu.click` | <ul><li>name</li><li>data-id</li><li>selection</li></ul>|
 
+### Diagram events
+
+| Event Name | Structure |
+| :--- | :--- |
+| `diagram.select`| <ul><li>name</li><li>old-selection</li><li>new-selection</li></ul>|
```

Comment:
It's pretty cumbersome to access kebab-case keys in JS, so I'd suggest to use camelCase:

```suggestion
| `diagram.select`| <ul><li>name</li><li>oldSelection</li><li>newSelection</li></ul>|
```

Reply:
The question is whether we really want to use the `data-` prefix in the canvas tracking module :). We could access the dataset instead:

```javascript
payload.action = element.dataset.action;
```

**Still, this is outside of scope of this PR.**

Regarding the selection payload, I believe that kebab-case keys will worsen the library consumer's experience. Notice how it makes it more difficult to use destructuring:

```javascript
const {
  'old-selection': oldSelection, // I never remember whether the key or the value is the name of the variable ;)
  'new-selection': newSelection
} = event.payload;

// vs

const {
  oldSelection,
  newSelection
} = event.payload;
```

Reply:
@nikku what do you think?